src/canvas-extractor/edfi_canvas_extractor/extract_facade.py

```diff
@@ -161,6 +254,17 @@ def _get_grades(arguments: MainArguments) -> None:
     )
 
 
+@retry(
+    retry_on_exceptions=(
+        IOError,
+        CanvasException,
+        RequestException,
+        socket.timeout,
+        socket.error,
```

Comment:
Don't forget about adding `ConnectionAbortedError`. Actually, that is a sub-class of `ConnectionError`. These are the base Python types, not library-specific types. In Schoology we are retrying on `ConnectionError`, so let's add that one here too.

Reply:
✔️